sbin/build.sh

```diff
@@ -105,19 +109,20 @@ checkingAndDownloadingFreetype()
     echo "Skipping FreeType download"
   else
     # Then FreeType for fonts: make it and use
-    wget -nc http://ftp.acc.umu.se/mirror/gnu.org/savannah/freetype/freetype-$FREETYPE_FONT_VERSION.tar.gz
+    wget -nc http://ftp.acc.umu.se/mirror/gnu.org/savannah/freetype/freetype-"$FREETYPE_FONT_VERSION".tar.gz
      
-    tar xvf freetype-$FREETYPE_FONT_VERSION.tar.gz
-    rm freetype-$FREETYPE_FONT_VERSION.tar.gz
+    tar xvf freetype-"$FREETYPE_FONT_VERSION".tar.gz
+    rm freetype-"$FREETYPE_FONT_VERSION".tar.gz
 
-    cd freetype-$FREETYPE_FONT_VERSION
+    cd freetype-"$FREETYPE_FONT_VERSION" || exit
 
-    if [ $(uname -m) = "ppc64le" ]; then
+    if [ "$OS_CPU_NAME" = "ppc64le" ]; then
+      # shellcheck disable=SC1083
       PARAMS="--build=$(rpm --eval %{_host})"
     fi
 
     # We get the files we need at $WORKING_DIR/installedfreetype
-    bash ./configure --prefix=$WORKING_DIR/$OPENJDK_REPO_NAME/installedfreetype $PARAMS && make all && make install
+    bash ./configure --prefix="$WORKING_DIR"/"$OPENJDK_REPO_NAME"/installedfreetype "$PARAMS" && make all && make install
```

Comment:
I think `$PARAMS`  is a single value, correct?  If not we should be using arrays.